### PR TITLE
Minor docs formatting

### PR DESCRIPTION
**Pull request by tlrobinson**

### CODE REVIEW

File: docs/operations-guide/running-metabase-on-heroku.md [hunk 1]
```diff
@@ -45,15 +45,21 @@ Here's each step:
 
 * Clone the latest version to your local machine:
 
+```bash
     git clone https://github.com/metabase/metabase-deploy.git  
     cd metabase-deploy
+```
```
Comment (camsaul) [inline docs/operations-guide/running-metabase-on-heroku.md]: This shouldn't stay indented if you add triple backticks, if you look at the rendered page it formats the contents as-is and includes all the extra space on the left margin

Comment (tlrobinson) [inline docs/operations-guide/running-metabase-on-heroku.md]: Good point, fixed.

**Comment by tlrobinson** [comment]
@metabase/core-developers How do I preview how the docs will look on the actual docs site?

**Comment by salsakran** [comment]
pull down the website's jekyll project, and copy the docs directory over to the `_docs/latest` directory

**Comment by salsakran** [comment]
it looks like the below image on my machine

screen shot 2016-11-04 at 2 14 27 pm

**Comment by tlrobinson** [comment]
Looks ok to you?

**Comment by salsakran** [comment]
Yup.

**Pull request merged**